windows/setup.ps1

```diff
@@ -155,6 +198,7 @@ if ($winrm_cert_expiry)
 
 $PendingReboot = Get-PendingRebootStatus
 Set-Attr $result.ansible_facts "ansible_reboot_pending" $PendingReboot
+Set-Attr $result.ansible_facts "module_setup" $true
```

Comment:
I'm curious, what is the purpose of module_setup.  If it is allways $true isn't it a bit redundant?